src/data/messages.py

```diff
@@ -1,68 +1,69 @@
-msg_dict = {
```

Comment:
Why this change exist? Fix the line endings  and force push

Reply:
force push to main?